From ba187d95c25905b5966dd0993afe45adbe07b710 Mon Sep 17 00:00:00 2001
From: Alex Deucher <alexdeucher@gmail.com>
Date: Mon, 7 Jun 2010 02:24:54 -0400
Patch-mainline: 2.6.35
References: fate#310916
Git-commit: cbd4623d4d3a622de6481052b44cd33ea880cd61
Subject: [PATCH 0849/2588] drm/radeon/kms: fix typo in printing the HPD info

I forgot to fix this in 8e36ed00842668a39a6ed1b0a00b8ac92b7c4cd5

Signed-off-by: Alex Deucher <alexdeucher@gmail.com>
Signed-off-by: Dave Airlie <airlied@redhat.com>
(cherry picked from commit cbd4623d4d3a622de6481052b44cd33ea880cd61)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/radeon/radeon_display.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/radeon/radeon_display.c b/drivers/gpu/drm/radeon/radeon_display.c
index c9080f4..eb3d18b 100644
--- a/drivers/gpu/drm/radeon/radeon_display.c
+++ b/drivers/gpu/drm/radeon/radeon_display.c
@@ -284,8 +284,7 @@ static const char *connector_names[15] = {
 	"eDP",
 };
 
-static const char *hpd_names[7] = {
-	"NONE",
+static const char *hpd_names[6] = {
 	"HPD1",
 	"HPD2",
 	"HPD3",
-- 
1.7.6

